doc/content/verification_and_validation/val-2f.md

```diff
@@ -0,0 +1,209 @@
+| $\epsilon_r^5$ | Release energy for trap 5     | 2.05                                                        | eV                    | [!cite](dark2024modelling) |
+| $\epsilon_r^6$ | Release energy for intrinsic trap     | 1.04                                                        | eV                    | [!cite](dark2024modelling) |
+| $N_1$ | Density for trap 1                          | 4.8$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_2$ | Density for trap 2                             | 3.8$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_3$ | Density for trap 3                             | 2.6$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_4$ | Density for trap 4                             | 3.6$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_5$ | Density for trap 5                             | 1.1$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_6$ | Density for intrinsic trap                             | 2.4$\times 10^{22}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+
+## Results
+
+[val-2f_comparison] shows the comparison of the TMAP8 calculation and the experimental data during desorption. The experimental data are provided by T. Schwarz-Selinger and are available [here](https://zenodo.org/records/11085134).
+
+!media comparison_val-2f.py
+       image_name=val-2f_comparison.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=val-2f_comparison
+       caption=Comparison of TMAP8 calculations with experimental data on deuterium flux (atoms/m$^2$/s) for a damage of 0.1 dpa and a normal recombination rate.
+
+[val-2f_deuterium_desorption] displays the quantities of mobile, trapped, and desorbing deuterium atoms during the desorption process. During desorption, the temperature increases from 300 K to 1000 K. The amount of deuterium trapped will decrease as the temperature rises and the various trapping energies are reached, meaning that deuterium will leave the traps, become mobile, and diffuse out. During desorption, no further implantation occurs, resulting in a decrease in the number of mobile and trapped deuterium atoms and an increase in the number of desorbed deuterium atoms. Mass conservation is well maintained during desorption, with only a 0.02% root mean squared percentage error (RMSPE) between the initial number of mobile and trapped deuterium atoms and the total number of deuterium atoms (mobile, trapped, and desorbed).
+
+!media comparison_val-2f.py
+       image_name=val-2f_deuterium_desorption.png
```

Comment:
Please do as we discussed and organize the legend in the same order as the data. This is currently reversed. 

Also, the color gradient is not varied anough and it's hard to know what is what. Use a different color scheme. 
This comment is valid for the other figures below.